### ISSUE TITLE
Changes about the crafting of Diamond plates

### ISSUE THREAD

**Issue by MadRatSRP** [GTNewHorizons/GT-New-Horizons-Modpack#10031]
### Your GTNH Discord Username

MadRat96#9831

### Your Pack Version

2.1.2.1

### Your Proposal

I wanted to craft diamond chest, but for some reason its not craftable up until getting Cutting Machine. 
![gt_enhancement_2](https://user-images.githubusercontent.com/29717217/160108841-0e2445c7-0bc7-43f6-a479-fa5ff2d5c8ac.png)

I think GTNH should either provide a Steel Chest upgrade, which would take the place of Diamond chest and diamond chest could be buffed up to 15x9 or 
1)There should be a way to craft Diamond plates in steam age, because you're able to make Diamond Screws, which are made from the flawless diamond. It could be uneffective, but still
2)Or if its made because of the balance (so player won't be able to make diamond armor for example) there should be used another thing instead of Diamond plate

### Your Goal

null

### Your Vision

null

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Dream-Master** [comment]
I see no need for diamond plates in steam age. If there are screw recipes they need to removed and put in machines only. Gold chest are big enough.

**Comment by Dream-Master** [comment]
Not sure if this is easy and possible. Never work with inventory slots.

**Comment by TwoTailedFox** [comment]
I'd probably create a new line of chests above Diamond with more slots with metals reflecting tier progression, i.e. Diamond/Obsidian/Crystal -> Aluminium -> Stainless Steel -> Tungstensteel ->Naquadah -> Naquadah Alloy -> Neutronium, etc

**Comment by Chaoschaot232** [comment]
> I'd probably create a new line of chests above Diamond with more slots with metals reflecting tier progression, i.e. Diamond/Obsidian/Crystal -> Aluminium -> Stainless Steel -> Tungstensteel ->Naquadah -> Naquadah Alloy -> Neutronium, etc

It is not a bad idea but this would make drawers/barrels kindly obsolet from the start on. And - even if it's end game content - there is the infinity chest.

![image](https://user-images.githubusercontent.com/1703441/160147341-dce592be-8de4-4f6d-a2c5-e16075aee8e0.png)

This chest has a rly ... uhm ... interesting recipie so far but tbh. it stores huge amount of items as long as the NBT doesn't get's disrupted ^^

So I would leave it as it is for now and don't add more types of chests as you can later make an ME System anyways.

@Dream-Master - From ym understanding you would first need the correct GUI and then simply copy over the code from the Diamond Chest and change stuff in that class(es) there, where they needs to be done without geting headaches. Because why creating new stuff it it's already there and could be used and extended🤔

**Comment by Ngar1** [comment]
Sorry, but I do not see any need for a diamond chest or diamond plates in steam age. 
That diamond screws are possible to craft is not a real argument for a diamond chest.
The LV age is not so far from steam age - just view RL days

**Comment by Glease** [comment]
I agree with dream here. IMO gold chests are big enough for your steam age endeavors. I don't see why you'd be running out of storage unless you are keeping every stack of cobble you ever mined.

**Comment by MadRatSRP** [comment]
> I agree with dream here. IMO gold chests are big enough for your steam age endeavors. I don't see why you'd be running out of storage unless you are keeping every stack of cobble you ever mined.

Ore dusts take the whole chest

**Comment by Ngar1** [comment]
make more chests ;-)                                like every one

**Comment by MadRatSRP** [comment]
I agree with your positions about that, but still maybe it should be revised a bit, not in terms of diamond plates but in terms of iron chests progression. For example, make the steel the highest available upgrade in steam age instead of gold (in terms of inventory's height progression) and later materials + diamond could be used for iron chests's inventory width upgrades, what do you think about that?

**Comment by Dream-Master** [comment]
> > I agree with dream here. IMO gold chests are big enough for your steam age endeavors. I don't see why you'd be running out of storage unless you are keeping every stack of cobble you ever mined.
> 
> Ore dusts take the whole chest

craft 2 or more
